src/Libraries/CoreNodeModels/Properties/Resources.resx

```diff
@@ -584,40 +584,4 @@
   <data name="SelectionPointsOutputPortName" xml:space="preserve">
     <value>Points</value>
   </data>
-  <data name="ListGroupByKeyDescription" xml:space="preserve">
-    <value>Group items into sub-lists based on their like key values</value>
```

Comment:
sorry I missed this the first time around - is there a reason you have not added all of these descriptions and tooltips to the zero touch nodes?